spec/controllers/api/v1/subjects_controller_spec.rb

```diff
@@ -209,6 +213,32 @@
             expect(retired).to all( be false )
           end
 
+          describe "favorited subjects" do
+            context "user has no favorites collection" do
+              it 'should return favorite as false' do
+                user.collections.destroy_all
```

Comment:
Any reason why we pay the cost of making collections only to destroy them? Best just to remove the auto-create ! in let and call them when (before block) you need them.